src/handwritten/stream_readers.py

```diff
@@ -238,6 +238,79 @@ def read_one_sample(self, timeout=10):
         """
         return self._interpreter.read_analog_scalar_f64(self._handle, timeout)
 
+    def read_waveform(
+        self, 
+        number_of_samples_per_channel: int = READ_ALL_AVAILABLE, 
+        timeout: int = 10, 
+        waveform: AnalogWaveform[numpy.float64] | None = None
+    ) -> AnalogWaveform[numpy.float64]:
+        """
+        Reads a waveform from a single analog input channel in a task.
+
+        Args:
+            number_of_samples_per_channel (Optional[int]): Specifies the
+                number of samples to read.
+
+                If you set this input to nidaqmx.constants.
+                READ_ALL_AVAILABLE, NI-DAQmx determines how many samples
+                to read based on if the task acquires samples
+                continuously or acquires a finite number of samples.
+
+                If the task acquires samples continuously and you set
+                this input to nidaqmx.constants.READ_ALL_AVAILABLE, this
+                method reads all the samples currently available in the
+                buffer.
+
+                If the task acquires a finite number of samples and you
+                set this input to nidaqmx.constants.READ_ALL_AVAILABLE,
+                the method waits for the task to acquire all requested
+                samples, then reads those samples. If you set the
+                "read_all_avail_samp" property to True, the method reads
+                the samples currently available in the buffer and does
+                not wait for the task to acquire all requested samples.
+            timeout (Optional[float]): Specifies the amount of time in
+                seconds to wait for samples to become available. If the
+                time elapses, the method returns an error and any
+                samples read before the timeout elapsed. The default
+                timeout is 10 seconds. If you set timeout to
+                nidaqmx.constants.WAIT_INFINITELY, the method waits
+                indefinitely. If you set timeout to 0, the method tries
+                once to read the requested samples and returns an error
+                if it is unable to.
+            waveform (Optional[AnalogWaveform]): Specifies an existing
+                AnalogWaveform object to use for reading samples. If
+                provided, the raw_data array of this waveform will be
+                used to store the samples. If not provided, a new
+                AnalogWaveform will be created.
```

Comment:
Use the word "into"

https://github.com/ni/nidaqmx-python/blob/d7ab6111fb4dd20502a7b64e322f9f089068b0ba/src/codegen/templates/task/_in_stream.py.mako#L280